config/locales/en.yml

```diff
@@ -762,9 +762,7 @@ en:
           label: Add a note
           submit: Save
         outbound_call_synthetic_note: Called by %{user_name}. Call was %{status} and lasted %{duration}.
-        outbound_call_synthetic_note_body: 'Call notes:
```

Comment:
@jenny-heath @alaroia-cfa  I did a rebase because I encountered some merge conflicts. Saw a lot of YML with this extra line in between the last quote. Assuming that this was an issue with changing the translations and should be removed? Let me know if this is okay to merge, thanks!